operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -57,7 +70,29 @@ type testGroup []struct {
 	outputDir                   string
 	diffSelect                  string
 	diffIgnore                  string
-	useCompiledInCharts         bool
+	chartSource                 chartSourceType
+}
+
+// TestMain is required to create a local release package in /tmp from manifests and operator/data in the format that
+// istioctl expects.
+func TestMain(m *testing.M) {
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	operatorRootDir = filepath.Join(wd, "../..")
+	manifestsDir = filepath.Join(operatorRootDir, "manifests")
+	liveReleaseDir, err = createLocalReleaseCharts()
```

Comment:
If the tests have to do this don't the users (maybe just devs) also have to do this? Why not just move the profiles in the manifests/ folder? is that all that is required

Reply:
yeah, that's all that's needed. we do it as part of the release for users but for devs it will save work.